SCClassLibrary/Common/Quarks/QuarksGui.sc

```diff
@@ -378,7 +378,9 @@ QuarkDetailView {
 		});
 	}
 	openGitRemote {
-		model.git.remoteAsHttpUrl.openOS;
+		if (model.git.notNil, {
+			model.git.remoteAsHttpUrl.openOS;
+		}, { model.url.openOS });
```

Comment:
Our (WIP) code style guide doesn't say anything about how to write If statements. Also, this file doesn't have a consistent style for you to follow. Most people here seems to prefer ```if(cond) { trueBlock } { falseBlock }```, with true and false branches broken into multiple lines. At a minimum, you could break the false branch into multiple lines and keep the rest as is.

Reply:
usually, we keep it consistent over a document. But there is a lot to do to adjust these to a more consistent standard.

Reply:
@patrickdupuis I quickly counted 42 `if` structures in that file and while 5 use the notation you are requesting, 37 use `if(..., { ... }, { ... })` so I'd vote for keeping it like that for now so there is more consistency in that file.

Reply:
Since there is neither an official code style, nor any form of linting to guide such decisions  I took the majority vote in this file. Just look at the method 4 lines above.

To lessen the burden on the maintainers of this repo it might be a good idea to either prioritize the code style or ignore such concerns for the time being to prevent bike shedding.